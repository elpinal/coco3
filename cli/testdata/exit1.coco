echo 'starting exit1!'
echo 'exiting 42!'
exit 42
