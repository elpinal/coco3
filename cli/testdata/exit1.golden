starting exit1!
exiting 42!
